Configure

```diff
@@ -1329,17 +1334,23 @@ if ($strict_warnings)
 	{
 	my $wopt;
 	my $gccver = $predefined{__GNUC__} // -1;
+	my $extra_gcc_warn = undef;
 
 	die "ERROR --strict-warnings requires gcc[>=4] or gcc-alike"
             unless $gccver >= 4;
-	$gcc_devteam_warn .= " -Wmisleading-indentation" if $gccver >= 6;
-	foreach $wopt (split /\s+/, $gcc_devteam_warn)
+	$extra_gcc_warn = "-Wmisleading-indentation" if $gccver >= 6;
+	foreach $wopt (split /\s+/, $devteam_common_warn)
+		{
+		$config{cflags} .= " $wopt" unless ($config{cflags} =~ /(?:^|\s)$wopt(?:\s|$)/)
```

Comment:
This line is actually too long. While you're at it, would it be appropriate to break it?

Reply:
On side note, there seems to be a bug in user_cflags handling. If you follow $no_user_flags [and $no_user_defines] you'll note that they are used before they are set to 1.

Reply:
I'll have a look at the `$user_flags` handling.  That'll be for a separate PR, though.

I'll try to break things up a bit better.